armada_jupyter/constants.py

```diff
@@ -25,3 +27,13 @@ class YMLSTR:
 DISABLE_SSL = os.environ.get("DISABLE_SSL", False)
 HOST = os.environ.get("ARMADA_SERVER", "localhost")
 PORT = os.environ.get("ARMADA_PORT", "50051")
+
+JOB_SET_ID = "armada-jupyter-pods"
```

Comment:
IMO, I think having this as a constant is wrong. Any user should need to specify a job set.  We don’t want to encourage different users to submit to same job set.  Is it required on submission?  

If it is, then we shouldn’t have this constant.

Reply:
Even better!